ledger/accountdb.go

```diff
@@ -92,6 +92,9 @@ var accountsSchema = []string{
 		id string primary key,
 		intval integer,
 		strval text)`,
+	`CREATE TABLE IF NOT EXISTS onlineroundparamstail(
```

Comment:
format like statements above. Btw, `accountsSchema` is schema v0. So this table should not be here, but in schema v6 to v7 upgrade